server/src/test/kotlin/core/gameflow/PlayerTest.kt

```diff
@@ -0,0 +1,55 @@
+package core.gameflow
+
+import core.Card
+import core.CardRank
+import core.CardSuit
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.assertThrows
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class PlayerTest {
+
+    private val testHoleCards: List<Card> = listOf(
```

Comment:
I would avoid explicitly specifying types when they can be easily deduced from the context. Kotlin's type inference mechanism will do it's job.